ros2bag/ros2bag/verb/record.py

```diff
@@ -34,168 +34,177 @@
+        next(iter(writer_choices))
+
+    serialization_choices = get_registered_serializers()
+    converter_suffix = '_converter'
+    serialization_choices = {
+        f[:-len(converter_suffix)]
+        for f in serialization_choices
+        if f.endswith(converter_suffix)
+    }
+
+    # Base output
+    parser.add_argument(
+        '-o', '--output',
+        help='Destination of the bagfile to create, '
+             'defaults to a timestamped folder in the current directory.')
+    parser.add_argument(
+        '-s', '--storage', default=default_writer, choices=writer_choices,
+        help="Storage identifier to be used, defaults to '%(default)s'.")
+
+    # Topic filter arguments
+    topics_args_group = parser.add_mutually_exclusive_group()
+    topics_args_group.add_argument(
+        'topics_positional', type=str, default=[], metavar='[Topic ...]', nargs='*',
+        help='Space-delimited list of topics to record.')
+    topics_args_group.add_argument(
+        '--topics', type=str, default=[], metavar='Topic', nargs='+',
+        help='Space-delimited list of topics to record.')
+    parser.add_argument(
+        '--topic-types', nargs='+', default=[], help='List of topic types to record.')
+    parser.add_argument(
+        '-a', '--all', action='store_true',
+        help='Record all topics and services (Exclude hidden topic).')
+    parser.add_argument(
+        '--all-topics', action='store_true',
+        help='Record all topics (Exclude hidden topic).')
+    parser.add_argument(
+        '--all-services', action='store_true',
+        help='Record all services via service event topics.')
+    parser.add_argument(
+        '-e', '--regex', default='',
+        help='Record only topics and services containing provided regular expression. '
+             'Overrides --all, --all-topics and --all-services, applies on top of '
+             'topics list and service list.')
+    parser.add_argument(
+        '--exclude-regex', default='',
+        help='Exclude topics and services containing provided regular expression. '
+             'Works on top of --all, --all-topics, or --regex.')
+    parser.add_argument(
+        '--exclude-topic-types', type=str, default=[], metavar='ExcludeTypes', nargs='+',
+        help='List of topic types not being recorded. '
+             'Works on top of --all, --all-topics, or --regex.')
+    parser.add_argument(
+        '--exclude-topics', type=str, metavar='Topic', nargs='+',
+        help='List of topics not being recorded. '
+             'Works on top of --all, --all-topics, or --regex.')
+    parser.add_argument(
+        '--exclude-services', type=str, metavar='ServiceName', nargs='+',
+        help='List of services not being recorded. '
+             'Works on top of --all, --all-services, or --regex.')
+
+    # Enable to record service
+    parser.add_argument(
+        '--services', type=str, metavar='ServiceName', nargs='+',
+        help='List of services to record.')
```

Comment:
`List of ...` -> `Space-delimited list of`

Reply:
Fixed in the new commit https://github.com/ros2/rosbag2/pull/1632/commits/875e7876b45c676b3ebcd14a69abf402e94aa796